frontend/src/utils/user.ts

```diff
@@ -0,0 +1,37 @@
+export default class User {
+  checkedFileTypeContribution : number;
+
+  commits: Array<object>;
+
+  dailyCommits: Array<object>;
+
+  displayName: string;
+
+  fileTypeContribution: object;
```

Comment:
I was thinking about not having any `object` type here but I think it can be done another PR.

Reply:
I believe `fileTypeContribution` had nested objects so I did not implement them yet. Will work on it later.